interface/patient_file/summary/add_edit_amendments.php

```diff
@@ -184,13 +184,8 @@ function formValidation() {
 
                     <div class="form-group mt-3">
                         <label><?php echo xlt('Request Description'); ?></label>
-                        <textarea <?php echo ( $onlyRead ) ? "readonly" : "";  ?> id="desc" class="form-control" name="desc" rows="4" cols="30"><?php
-                        if ($amendment_id) {
-                            echo text($amendment_desc);
-                        } else {
-                            echo "";
-                        } ?>
-                        </textarea>
+                        <textarea <?php echo ( $onlyRead ) ? "readonly" : "";  ?> id="desc" class="form-control" name="desc" rows="4" cols="30"
+                            placeholder="<?php echo $amendment_id ? text($amendment_desc) : ""; ?>"></textarea>
```

Comment:
Wait...this actually fills values...my bad. Need to revert this.

Reply:
@tywrenn OK, revert the commit.